changelogs/unreleased/598-carthewd

```diff
@@ -0,0 +1 @@
+enhancement: allow option to choose JSON log output 
```

Comment:
we usually name this file based on the PR #, so let's change it to `1654-carthewd`